discovery-provider/src/challenges/challenges.json

```diff
@@ -20,5 +20,18 @@
 		"active": false,
 		"step_count": 3,
 		"starting_block": 30000000
+	},
+	{
+		"id": "referrals",
+		"type": "aggregate",
+		"amount": "5",
```

Comment:
TODO: Confirm these values

Reply:
always subject to change, but afaik 1 for now